code/game/objects/items/busterarm.dm

```diff
@@ -0,0 +1,913 @@
+	user.Immobilize(0.2 SECONDS)//so it's not cut short by walking
+	user.throw_at(get_step_towards(target,user), 8, 4)
+
+/obj/item/projectile/wire/on_hit(atom/target)
+	var/mob/living/carbon/human/H = firer
+	if(isobj(target))
+		var/obj/item/I = target
+		if(!I?.anchored)
+			I.throw_at(get_step_towards(H,I), 8, 2)
+			H.visible_message(span_danger("[I] is pulled by [H]'s wire!"))
+			if(istype(I, /obj/item/clothing/head))
+				H.equip_to_slot_if_possible(I, SLOT_HEAD)
+				H.visible_message(span_danger("[H] pulls [I] onto [H.p_their()] head!"))
+			else
+				H.put_in_hands(I)
+			return
+		zip(H, target)
+	if(isliving(target))
+		var/mob/living/L = target
+		var/turf/T = get_step(get_turf(H), H.dir)
+		var/turf/Q = get_turf(H)
+		var/obj/item/bodypart/limb_to_hit = L.get_bodypart(H.zone_selected)
+		var/armor = L.run_armor_check(limb_to_hit, MELEE, armour_penetration = 35)
+		if(!L.anchored)
+			if(istype(H))
+				L.visible_message(span_danger("[L] is pulled by [H]'s wire!"),span_userdanger("A wire grabs you and pulls you towards [H]!"))				
+				L.Immobilize(1.5 SECONDS)
+				if(prob(5))
+					firer.say("GET OVER HERE!!")//slicer's request
+				if(T.density)
+					to_chat(H, span_warning("[H] catches [L] and throws [L.p_them()] against [T]!"))
+					to_chat(L, span_userdanger("[H] crushes you against [T]!"))
+					playsound(L,'sound/effects/pop_expl.ogg', 130, 1)
+					L.apply_damage(15, BRUTE, limb_to_hit, armor, wound_bonus=CANT_WOUND)
+					L.forceMove(Q)
+					return
+				for(var/obj/D in T.contents)
+					if(D.density == TRUE)
+						D.take_damage(50)	
+						L.apply_damage(15, BRUTE, limb_to_hit, armor, wound_bonus=CANT_WOUND)
+						L.forceMove(Q)
+						to_chat(H, span_warning("[H] catches [L] throws [L.p_them()] against [D]!"))
+						playsound(L,'sound/effects/pop_expl.ogg', 20, 1)
+						return
+				L.forceMove(T)
+	if(iswallturf(target))
+		var/turf/W = target
+		zip(H, W)
+
+/obj/item/gun/magic/wire/process_chamber()
+	. = ..()
+	if(charges == 0)
+		qdel(src)
+
+/obj/item/projectile/wire/Destroy()
+	qdel(wire)
+	return ..()
+
+/obj/effect/proc_holder/spell/targeted/touch/buster/grap
+	name = "Grapple"
+	desc = "Prepare your left hand for grabbing. Throw your target and inflict more damage if they hit a solid object. If the targeted limb is horribly bruised, you'll tear it off\
+	when throwing the victim."
+	hand_path = /obj/item/buster/graphand
+	school = "transmutation"
+	charge_max = 30
+	action_icon = 'icons/mob/actions/actions_arm.dmi'
+	action_icon_state = "lariat"
+	sound = 'sound/magic/fleshtostone.ogg'
+	clothes_req = FALSE
+	var/jumpdistance = 4
+
+/obj/effect/proc_holder/spell/targeted/touch/buster/grap/cast(list/targets, mob/user = usr)
+	if(!QDELETED(attached_hand))
+		remove_hand(TRUE)
+		to_chat(user, span_notice("[dropmessage]"))
+		return
+	for(var/mob/living/carbon/C in targets)
+		C.visible_message(span_warning("The fingers on [C]'s left buster arm begin to tense up."))
```

Comment:
if the arm can be on either side of the body, why the left arm?

Reply:
ah fuck missed that thanks

Reply:
> if the arm can be on either side of the body, why the left arm?

wait no disregard what i said before this is cause this spell is specifically for the corresponding arm hence the other version existing